src/get-started/timestamping-tutorial.md

```diff
@@ -879,13 +880,13 @@ performed in the directory containing `Cargo.toml`.
 
    ```shell
    exonum-demo-timestamping generate-config example/common.toml \
-       example/pub_1.toml example/sec_1.toml --peer-address 127.0.0.1:6331
+       example/pub_1.toml example/sec_1.toml --peer-address 127.0.0.1:6331 -n
```

Comment:
What is `-n` in the end of the line ? I can't remember that we have a such parameter.